harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
There is an implementation of test isolation in the browser, it's the old test262 harness, though it can't run ES6 tests. The isolation is not hard - you just use iframes. Yes, these destructive helpers make it harder to write tests, but what it forces you to do is good - think very hard about side effects, and test different things in different files. Test262 is not a typical test harness - its high order bit is not convenience for test authors but instead value for implementations (especially half-done or buggy ones).

IMO this should be closed.